app/models/case_log.rb

```diff
@@ -567,13 +566,19 @@ def age_under_16?(person_num)
     public_send("age#{person_num}") && public_send("age#{person_num}") < 16
   end
 
+  def format_postcode(value)
```

Comment:
I think this probably doesn't really belong in the case log model/class since we display postcodes in multiple places (e.g. scheme locations).

There's a postcode service class that would be a better fit, but take a look at how we do date formatting for display https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/config/initializers/date_formats.rb